### PR TITLE
8280172: Create release notes for JavaFX 17.0.2

### PR DESCRIPTION
**Pull request by abhinayagarwal**
Release Notes for 17.0.2

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8280172](https://bugs.openjdk.java.net/browse/JDK-8280172): Create release notes for JavaFX 17.0.2

### Reviewers
 * [Kevin Rushforth](https://openjdk.java.net/census#kcr) (@kevinrushforth - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jfx17u pull/28/head:pull/28` \
`$ git checkout pull/28`

Update a local copy of the PR: \
`$ git checkout pull/28` \
`$ git pull https://git.openjdk.java.net/jfx17u pull/28/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 28`

View PR using the GUI difftool: \
`$ git pr show -t 28`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jfx17u/pull/28.diff">https://git.openjdk.java.net/jfx17u/pull/28.diff</a>

</details>

### CODE REVIEW

File: doc-files/release-notes-17.0.2.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+JavaFX 17.0.2 requires JDK 11 or later.
+
+List of Task
+------------
+
+| Issue Key                                                       | Summary                                                 | Subcomponent |
+| --------------------------------------------------------------- | ------------------------------------------------------- | ------------ |
+| [JDK-8272638](https://bugs.openjdk.java.net/browse/JDK-8272638) | Update copyright header for files modified in 2021      | other        |
+| [JDK-8274413](https://bugs.openjdk.java.net/browse/JDK-8274413) | FX: Update copyright year in docs, readme files to 2022 | other        |
+| [JDK-8279396](https://bugs.openjdk.java.net/browse/JDK-8279396) | Define version in .jcheck/conf                          | other        |
```
Comment (abhinayagarwal) [inline doc-files/release-notes-17.0.2.md:18]: Hi @kevinrushforth ,

Can you confirm if issues like this needs to be in the release note. The issue itself is labelled as `noreg-cleanup`, however, the backport issue for openjfx17.0.1 doesn't have any such label.

File: doc-files/release-notes-17.0.2.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+JavaFX 17.0.2 requires JDK 11 or later.
+
+## List of Task
+
+Issue key|Summary|Subcomponent
+---------|-------|------------
+[JDK-8272638](https://bugs.openjdk.java.net/browse/JDK-8272638)|Update copyright header for files modified in 2021|other
+[JDK-8274413](https://bugs.openjdk.java.net/browse/JDK-8274413)|FX: Update copyright year in docs, readme files to 2022|other
+[JDK-8279396](https://bugs.openjdk.java.net/browse/JDK-8279396)|Define version in .jcheck/conf|other
```
Comment (kevinrushforth) [inline doc-files/release-notes-17.0.2.md:17]: We normally exclude `Task`s along with build and test changes from the release notes.

**Review by kevinrushforth — COMMENTED** [review]
The list of bugs fixed looks correct, with the addition of the following security fix:

```
JDK-8272546 (not public) | Better TrueType font loading | graphics
```

See [release-notes-17.md](https://github.com/openjdk/jfx17u/blob/master/doc-files/release-notes-17.md#list-of-security-fixes) for an example of how we include security fixes.

Additionally, one comment left inline below.

File: doc-files/release-notes-17.0.2.md (hunk 2 above)
Comment (kevinrushforth) [reply doc-files/release-notes-17.0.2.md:17]: I don't see a comment asking about it, but yes, your understanding is correct. The release notes should include bugs and enhancements (or backports of same) without build or test labels.

**Comment by kevinrushforth** [comment]
The latest addition looks good. Per the above discussion, you should remove the "List of Task" section.

**Review by kevinrushforth — APPROVED** [review]

**Pull request closed without merging**